src/Entity/DatasetSubmission.php

```diff
@@ -1028,6 +1028,15 @@ class DatasetSubmission extends Entity
      */
     protected $remotelyHostedFunction;
 
+    /**
+     * The Point of Contact for this Dataset Submission.
+     *
+     * @var Collection
+     *
+     * @ORM\OneToMany(targetEntity="App\Entity\DatasetLink", mappedBy="datasetSubmission", cascade={"persist"}, orphanRemoval=true)
+     */
+    private $datasetLinks;
```

Comment:
private vs protected?

Reply:
This was added by the Maker bundle, but changed it.